test/api-digester/stability-stdlib-abi-with-asserts.test

```diff
@@ -56,6 +56,9 @@
 Func _collectReferencesInsideObject(_:) is a new API without @available attribute
 Func _loadDestroyTLSCounter() is a new API without @available attribute
 Func _measureRuntimeFunctionCountersDiffs(objects:_:) is a new API without @available attribute
+Func withUnsafeUninitializedMutableBufferPointer(to:capacity:_:) is a new API without @available attribute
+Func withUnsafeUninitializedMutablePointer(to:_:) is a new API without @available attribute
+Func withUnsafeUninitializedMutableRawBufferPointer(byteCount:alignment:_:) is a new API without @available attribute
```

Comment:
API changes should normally be listed in the "stability-stdlib-abi-***without***-asserts.test" file instead.

https://github.com/apple/swift/blob/2950c38a5d4f1f028d3786a278ae61eaf501f0d0/test/api-digester/stability-stdlib-abi-with-asserts.test#L14-L17

However, your new APIs probably need one of:

* [`@_alwaysEmitIntoClient`](<https://github.com/apple/swift/blob/main/docs/ReferenceGuides/UnderscoredAttributes.md#_alwaysemitintoclient>) if possible; otherwise

* [`@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)`](<https://github.com/apple/swift/blob/main/docs/StandardLibraryProgrammersManual.md#availability>)

Reply:
@benrimmington You're probably right—I need to check with @jckarter and/or the other core team members about the right annotation. I'm following the precedent set by other inlinable pointer functions here.

Reply:
Fixed locally, pushing a change to use `@_aEIC` and remove the change to this test.